io.openliberty.java.internal_fat_misc/java17/build.gradle

```diff
@@ -0,0 +1,142 @@
+        ]
+        classpath = files()
+        options.warnings = true
+        options.deprecation = true
+        options.debug = true
+        options.incremental = false
+    }
+}
+test {
+  useJUnitPlatform()
+  jvmArgs '--add-modules=jdk.incubator.foreign'
+}
+
+java {
+  toolchain {
+    languageVersion = JavaLanguageVersion.of(17)
+  }
+}
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'io.openliberty.tools:liberty-gradle-plugin:3.0'
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+configurations {
+    serverLibs
+}
+
+dependencies {
+    compileOnly group: 'javax.ws.rs', name: 'javax.ws.rs-api', version: '2.1.1'
+//    compileOnly group: 'javax.persistence', name: 'javax.persistence-api', version: '2.2'
+    compileOnly group: 'javax.xml.bind', name: 'jaxb-api', version: '2.3.1'
+    compileOnly group: 'javax.enterprise', name: 'cdi-api', version: '2.0'
+//    compileOnly group: 'javax.annotation', name: 'javax.annotation-api', version: '1.3.2'
+//    compileOnly group: 'javax.transaction', name: 'javax.transaction-api', version: '1.3'
+    libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '21.0.0.9'
+//    serverLibs group: 'org.apache.derby', name: 'derby', version: '10.14.2.0' // '10.15.1.3'
+}
+
+ext {
+	appUrl = 'http://localhost:9080/java17-app/'
+}
+
+liberty {
+	server {
+		name = rootProject.name + 'Server'
+                deploy {
+                    dropins = [war]
+		}
+                stripVersion = true
+	}
+	// Default location of local Liberty install
+    installDir = "/wlp"
+}
+
+task copyServerLibs(type: Copy) {
+    from configurations.serverLibs
+    // Default location of local Liberty server
+    into "/wlp/usr/servers/${rootProject.name}Server/lib"
+}
+
+// Not currently working, must manually handle
+tasks.register('copyEnvFiles', Copy) {
+    dependsOn 'libertyStop'
+    println "Start of copyEnvFiles"
+    from file('run/jvm.options')
+    // Default location of local Liberty server
+    into new File("/wlp/usr/servers/${rootProject.name}Server")
+//    from file('run/server.env')
+    // Default location of local Liberty server
+//    into new File("/wlp/usr/servers/${rootProject.name}Server")
+    println "End of copyEnvFiles"
+}
+
+task openBrowser {
+    description = "Open browser to ${appUrl}"
+    doLast {
+        java.awt.Desktop.desktop.browse "${appUrl}".toURI()
+    }
+}
+
+deploy.dependsOn 'copyServerLibs'
+libertyPackage.dependsOn 'libertyStop', 'copyServerLibs' 
+
+clean.dependsOn 'libertyStop'
+libertyPackage.dependsOn 'libertyStop'
+libertyStart.dependsOn 'libertyStop'
+libertyRun.dependsOn 'libertyStop'
+
+libertyStart.doFirst {
+	copyEnvFiles
+}
+
+libertyStart.doLast {
+	println "Application available at: ${appUrl}"
+}
+
+libertyStart.dependsOn 'libertyStop'
+libertyRun.dependsOn 'libertyStop'
+
+task start { dependsOn 'libertyStart' }
+task stop  { dependsOn 'libertyStop' }
+
```

Comment:
Are we going to be publishing this artifact?  
If so we would need a publishing directive within this build file: 
```
plugins {
    id 'maven-publish'
}
publishing {
    publications {
        maven(MavenPublication) {
            groupId 'io.openliberty.java'
            artifactId 'java-17-test-app'
            version '1.0'
            from components.java
        }
    }
}
```

Reply:
No plans to publish the generated WAR.